src/constants.js

```diff
@@ -6,5 +6,10 @@ export const KEY_CODES = {
 };
 
 export const ATTRIBUTES = {
-    UID: 'data-uid'
+    UID:     'data-uid',
+    NONCE:   'csp-nonce',
```

Comment:
```suggestion
    NONCE:   'data-csp-nonce',
```

Nonce is also prefixed with `data-` https://developer.paypal.com/docs/checkout/troubleshoot/support/#content-security-policy-rules

Reply:
Got it, changed.

Probably we should update the docs as well? This table is a bit misleading:

![image](https://user-images.githubusercontent.com/5726810/117829133-783f5680-b240-11eb-9dd7-5fd379e40306.png)


https://developer.paypal.com/docs/checkout/reference/customize-sdk/#csp-nonce